Android.mk

```diff
@@ -265,11 +265,19 @@ HYBRIS_UPDATER_UNPACK := $(LOCAL_BUILD_MODULE)
 
 .PHONY: hybris-hal hybris-common
 
-hybris-common: bootimage hybris-updater-unpack hybris-updater-script hybris-recovery hybris-boot servicemanager logcat updater init adb adbd linker libc libEGL libGLESv1_CM libGLESv2
+ifeq ($(shell test $(ANDROID_VERSION_MAJOR) -ge 8 && echo true),true)
+HYBRIS_COMMON_TARGETS := verity_signer boot_signer e2fsdroid vendorimage ramdisk bootimage hybris-updater-unpack hybris-updater-script hybris-recovery hybris-boot servicemanager logcat updater init adb adbd linker libc libEGL libGLESv1_CM libGLESv2 libsurfaceflinger libsf_compat_layer libselinux_stubs
```

Comment:
Could we instead introduce something like ```HYBRIS_ANDROID8_TARGETS := boot_signer e2fsdroid libselinux_stubs ramdisk vendorimage verity_signer``` and then reference it?

Reply:
@sledges sounds good, something like this? also i seperated the libsf_compat_layer part out, to make the issue with that one more noticeable